### PR TITLE
[chore] prepare release 0.18.0

### PR DESCRIPTION
**Pull request by bacherfl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,74 @@
...
+---
+
+</details>
+
+#### Dynatrace distribution changelog:
+
+### 💡 Enhancements 💡
+
+- `transformprocessor`: Extend the user data filtering example with a statement for masking the `client.address` attribute. (#356)
+- `transformprocessor`: Extend the credit card filtering example with log body filtering (#357)
```
Comment (mowies) [inline CHANGELOG.md:72]: [nitpick] I forgot to add that also IBAN log body filtering is included here

**Review by mowies — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bacherfl) [reply CHANGELOG.md]: thanks, I have adapted the not to also reflect that

**Review by mowies — APPROVED** [review]

**Comment by pirgeo** [comment]
Should we run `make gotidy` before releasing?

**Comment by bacherfl** [comment]
> Should we run `make gotidy` before releasing?

good catch, I will also add a note about this to the release check list so we don't forget it in the future

**Review by pirgeo — APPROVED** [review]

**Review by odubajDT — APPROVED** [review]

**Pull request merged**